CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
--  No changes yet.
+### Added
+-  Added a strucutred logger (zap) to internal logging.
```

Comment:
zap is an implementation detail. the user facing changelog should be higher level:

```suggestion
### Changed
-  Fx now emits structured, JSON logs. These may be parsed and processed by log ingestion systems.
```